plugins/chat/assets/javascripts/discourse/lib/chat-message-interactor.js

```diff
@@ -18,6 +18,12 @@ import ChatMessage from "discourse/plugins/chat/discourse/models/chat-message";
 import { MESSAGE_CONTEXT_THREAD } from "discourse/plugins/chat/discourse/components/chat-message";
 import I18n from "I18n";
 
+const removedSecondaryButtons = new Set();
+
+export function removeChatComposerSecondaryButtons(buttonIds) {
+  buttonIds.forEach((id) => removedSecondaryButtons.add(id));
+}
+
```

Comment:
This has the potential to leak from on test to another, we should make sure to reset it after writing a test for it.